src/main/java/Deadline.java

```diff
@@ -0,0 +1,21 @@
+public class Deadline extends Task {
+    protected String taskType;
+    protected String taskTime;
+
+    // When creating Deadline object, a Task object is created as well
+    // Hence either add empty constructor to Task, or use super in Deadline
+    public Deadline (String taskDescription, String taskTime) {
+        super(taskDescription);
+        this.taskType = "D";
+        this.taskTime = taskTime;
+    }
+
+    @Override
+    public void describe() {
```

Comment:
Perhaps could have a more descriptive method name? The name "describe" is not very clear to me at first glance